rumqttd/src/lib.rs

```diff
@@ -86,13 +86,14 @@ pub enum Error {
     #[error("Invalid server cert file {0}")]
     InvalidServerCert(String),
     #[error("Invalid server pass")]
-    InvalidServerPass(),
+    InvalidServerPass,
     #[error("Invalid server key file {0}")]
     InvalidServerKey(String),
     RustlsNotEnabled,
     NativeTlsNotEnabled,
     Disconnected,
     NetworkClosed,
+    #[error("Wrong packet {0:?}")]
```

Comment:
I agree that `text: {0}` is a better format for error messages, this could change here.